File: Semester - 03 OOP and Python Programming/OOP and Python Programming/Week - 01 Introduction to Python/Module - 02.5 Practice Day - 01/S_Sum_of_Consecutive_Odd_Numbers.py
def main():
    tc = int(input()) 
    for _ in range(tc):
        x, y = map(int, input().split())
        odd, a, b, i = 0, 0, 0, 0
        if x > y:
            a = y
            b = x
        else:
            a = x
            b = y
        res = a
        while a <= b:
            if a >= res and a <= b:
                if a % 2 != 0 and a != res and a != b:
                    odd += a
                a += 1
        print(odd)
if __name__ == "__main__":
    main()

File: Semester - 03 OOP and Python Programming/OOP and Python Programming/Week - 01 Introduction to Python/Module - 02.5 Practice Day - 01/S_Sum_of_Consecutive_Odd_Numbers.cpp
def sum_of_odd_numbers_between(x, y):
#Ensure x is always less than y
    if x > y:
        x, y = y, x

#Calculate the sum of odd numbers between x and y
    return sum(i for i in range(x+1, y) if i % 2 != 0)

def main():
#Read number of test cases
    T = int(input("Enter number of test cases: "))

#Process each test case
    for _ in range(T):
#Read the two numbers for each test case
        x, y = map(int, input("Enter two numbers separated by space: ").split())

#Print the sum of odd numbers between x and y
        print(sum_of_odd_numbers_between(x, y))

if __name__ == "__main__":
    main()
